src/accessibility-and-localization/internationalization.md

```diff
@@ -189,24 +190,26 @@ Widget build(BuildContext context) {
   );
 }
 ```
-Hot reload the app and observe that the `CalendarDatePicker` widget re-renders in Spanish.
+
+Hot reload the app and the `CalendarDatePicker`
+widget should re-render in Spanish.
 
 <a name="adding-localized-messages"></a>
 ### Adding your own localized messages
 
-After you add the `flutter_localizations` package, you can configure
-localization. To add localized text to your application, complete the
-following instructions.
+After adding the `flutter_localizations` package,
+you can configure localization.
+To add localized text to your application,
+complete the following instructions:
 
-1. Add the `intl` package as a dependency,
-   using the version pinned by `flutter_localizations` with `any`:
+1. Add the `intl` package as a dependency:
 
    ```terminal
-   $ flutter pub add intl:any
+   $ flutter pub add intl
```

Comment:
I think the `:any` portion was useful here as it is a dependency pinned by flutter/`flutter_localizations`, and apps should just pick up the dependency from there.